skbio/sequence/_grammared_sequence.py

```diff
@@ -122,6 +240,7 @@ def gap_chars(cls):
         """
         return set('-.')
```

Comment:
Can all abstract properties be defined with `pass` instead of sets? Since these are abstract we really shouldn't have defaults.